transpile/classes.py

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/python3
+
+class Block:
```

Comment:
This should be defined as an abstract base class (see the [docs](https://docs.python.org/3/library/abc.html))